packages/terra-overlay/src/Overlay.jsx

```diff
@@ -39,19 +40,24 @@ const propTypes = {
   * Indicates if the overlay is relative to the triggering container.
   */
   isRelativeToContainer: PropTypes.bool,
+  /**
+   * Allows assigning of root element custom data attribute for easy selecting.
```

Comment:
This documentation feels a little vague. What do we mean by the root element exactly? Root of the application, or root container that triggered the overlay to go.

Reply:
I changed the description a little bit in one of the more recent commits. The goal is to give an easy way to target whatever the user's version of the base component is... If the updated comment doesn't work, let me know what you suggest!